tooling-tests/src/test/resources/test-src/output/tool_test_generate_3/Ballerina.toml

```diff
@@ -5,7 +5,7 @@ version = "0.1.0"
 distribution = "2201.2.0"
 
 [build-options]
-observabilityIncluded = true
+observabilityIncluded = false
```

Comment:
Is this due to any change from out side?

Reply:
Yes. This doesn't affect the compilation but without this change we cannot generate .jar file, so `bal build` command fails and until this observability issue is fixed, we have to keep this disabled.